images/stackstorm/Dockerfile

```diff
@@ -81,6 +81,10 @@ RUN /install.sh --tag=${ST2_TAG} --st2=${ST2_VERSION} --st2web=${ST2WEB_VERSION}
 
 # Install chatops and disable unless entrypoint.d file is present
 # Install st2-chatops with Node.js v6/v10 requirement
+# This sorts (descending) a two-element list containing "3.0" and "${ST2_VERSION}",
+# using GNU sort's version comparison (-V).
+# If the "3.0" element is the first element, then $ST2_VERSION is greater than 3.0.
```

Comment:
I think you mean greater than or equal to `3.0`?

Reply:
Good catch!